src/utils/queries.ts

```diff
@@ -1,37 +1,51 @@
 /*
-
  MIT License
-
  Copyright (c) 2021 Looker Data Sciences, Inc.
-
  Permission is hereby granted, free of charge, to any person obtaining a copy
  of this software and associated documentation files (the "Software"), to deal
  in the Software without restriction, including without limitation the rights
  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
  copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions:
-
  The above copyright notice and this permission notice shall be included in all
  copies or substantial portions of the Software.
-
  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  SOFTWARE.
-
  */
 
-import {
+ import {
   ILookmlModel,
   ILookmlModelExplore,
   ILookmlModelExploreField
 } from '@looker/sdk/lib/sdk/3.1/models'
 import { exploreURL } from "./urls"
 import { Looker31SDK as LookerSDK } from '@looker/sdk/lib/sdk/3.1/methods'
-import { loadCached } from "./fetchers"
+
+const globalCache: any = {}
```

Comment:
Moved the globalCache stuff into Queries, which are the only user of it now, as the inline queries use a class based component. Anticipating work with <Visualization /> to replace this and get onto the `react-query` model

Reply:
Got it.